lib/jekyll/site.rb

```diff
@@ -432,6 +432,8 @@ def collections_path
     private
 
     def load_theme_configuration(config)
+      return config unless config["import_theme_config"] == true
```

Comment:
The logic here seemed a bit backwards and the variable name used here seems wrong based on the docs.

```suggestion
      return config if config["ignore_theme_config"] == true
```